packages/visx-xychart/src/components/XYChart.tsx

```diff
@@ -36,18 +43,52 @@ export type XYChartProps<
   xScale?: DataProviderProps<XScaleConfig, YScaleConfig>['xScale'];
   /** If DataContext is not available, XYChart will wrap itself in a DataProvider and set this as the yScale config. */
   yScale?: DataProviderProps<XScaleConfig, YScaleConfig>['yScale'];
+  /** Callback invoked for onPointerMove events for the nearest Datum to the PointerEvent _for each Series with pointerEvents={true}_. */
+  onPointerMove?: ({
+    datum,
+    distanceX,
+    distanceY,
+    event,
+    index,
+    key,
+    svgPoint,
+  }: PointerEventParams<Datum>) => void;
+  /** Callback invoked for onPointerOut events for the nearest Datum to the PointerEvent _for each Series with pointerEvents={true}_. */
+  onPointerOut?: (
+    /** The PointerEvent. */
+    event: React.PointerEvent,
+  ) => void;
+  /** Callback invoked for onPointerUp events for the nearest Datum to the PointerEvent _for each Series with pointerEvents={true}_. */
+  onPointerUp?: ({
+    datum,
+    distanceX,
+    distanceY,
+    event,
+    index,
+    key,
+    svgPoint,
+  }: PointerEventParams<Datum>) => void;
+  /** Whether to invoke PointerEvent handlers for all dataKeys, or the nearest dataKey. */
+  pointerEvents?: 'all' | 'nearest';
```

Comment:
maybe should be called `pointerEventDataKey` 🤔

Reply:
I guess this probably should mirror the implementation and could also support specific `dataKey | dataKey[]` as well